doc/src/manual/arrays.md

```diff
@@ -872,3 +872,22 @@ julia> r
  -1.20921  -0.383393
   0.0      -0.910506
 ```
+
+### Zero-dimensional arrays
+
+Zero-dimensional arrays are very similar to scalars, but there are important differences. The following line defines a zero-dimensional array:
```

Comment:
Maybe add some words about why zero dimensional arrays exists and why you need to mention them? E.g. they deserve a special mention because they are a special case which makes logical sense given the generic definition of arrays. Else it's a bit surprising to say what are their properties without even explaining that they exist first. It would also be useful to say upfront that you're talking about `Array{T,0}` objects.